packages/webpack-cli/lib/groups/AdvancedGroup.js

```diff
@@ -29,7 +31,22 @@ class AdvancedGroup extends GroupHelper {
             }
         }
         if (args.target) {
-            options.target = args.target;
+            const validTargets = [
+                'web',
+                'webworker',
+                'node',
+                'async-node',
+                'node-webkit',
+                'electron-main',
+                'electron-renderer',
+                'electron-preload',
+            ];
+            if (validTargets.includes(args.target)) {
+                options.target = args.target;
+            } else {
+                logger.warn(yellow(`"${args.target}" is an invalid value for "target" option. Using "web" by default.`));
```

Comment:
Yep we can't assume about the platform for which they;re bundling, throwing err and displaying the possible values would be ideal

Reply:
makes sense :+1: